start/software/software-transfer-via-htcondor-or-http.md

```diff
@@ -2,94 +2,60 @@
-
-	#!/bin/bash
-	# HTCondor will transfer the file for us. Uncomment the following if you prefer to transfer form the script
-	# wget --no-check-certificate http://stash.osgconnect.net/+username/words.tar.gz
-	tar xzf words.tar.gz
-	cat random_words | ./distribution
-
-Here is the submit file. Note that HTCondor allows to specify an URL as input file and it will download it for you. Substitute your user name for username in the `transfer_input_files` line:
-
-	########################
-	# Submit description file for short test program using http transfer
-	########################
-	Universe       = vanilla
-	Executable     = words.sh
-	# Uncomment the following line to use the remote executable
-	#transfer_executable = False
-	Error   = log/words.err.$(Cluster)-$(Process)
-	Output  = log/words.out.$(Cluster)-$(Process)
-	Log     = log/words.log.$(Cluster)
-	should_transfer_files = YES
-	transfer_input_files = http://stash.osgconnect.net/+username/words.tar.gz
-	when_to_transfer_output = ON_EXIT
-	Queue 5
-
-Submit the job:
-
-	$ condor_submit words.submit
-	Submitting job(s).....
-	5 job(s) submitted to cluster 14038.
-
-Once the jobs are completed, you can look at the output in the logs directory and verify that the job ran correctly:
-
-	$ cat log/words.out.14038-2
-	Ashkenazim |45 (0.44%) +++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	BIOS       |45 (0.44%) +++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Anaheim    |44 (0.43%) +++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Aymara     |44 (0.43%) +++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Arthurian  |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Anaxagoras |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Bactria    |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Alexis     |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Ariel      |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Aubrey     |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Baryshnikov|42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Bahia      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Angstrom   |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Asoka      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Alcatraz   |41 (0.40%) ++++++++++++++++++++++++++++++++++++++++++++++++++++
+For some OSG Connect users, it may be necessary or advatageous to install your own software. This guide will 
+describe steps and important considerations for transferring your installed software via the HTCondor submit file.  
 
+## Important Considerations
+
+How you transfer your software via HTCondor will depend on the size and location of the executable binary or tar archive 
+that needs to be transferred along with your jobs. Only software files <100MB staged in your `/home` directory or 

```

Comment:
I would simplify to "only software < 1GB should be..."
The later part of the guide talks about `/home` and `/public`.